Interfaces/API/Dirac.py

```diff
@@ -859,32 +859,33 @@ def runLocal(self, job):
       if isinstance( sandbox, basestring ):
         sandbox = [isFile.strip() for isFile in sandbox.split(',')]
       for isFile in sandbox:
-        self.log.debug("Resolving Input Sandbox file %s" % isFile)
+        self.log.debug("Resolving Input Sandbox %s" % isFile)
         if isFile.lower().startswith( "lfn:" ):  # isFile is an LFN
           isFile = isFile[4:]
-        elif not os.path.isabs( isFile ):
-          # if a relative path, it is relative to the user working directory
-          isFile = os.path.join( curDir, isFile )
-
-        # Attempt to copy into job working directory
-        if os.path.isdir( isFile ):
-          self.log.debug("Input Sandbox %s is a directory, found in the user working directory, copying it" % isFile)
-          shutil.copytree( isFile, os.path.basename( isFile ), symlinks = True )
-        elif os.path.exists( isFile ):
-          self.log.debug("Input Sandbox %s is a file, found in the user working directory, copying it" % isFile)
+        if os.path.isabs( isFile ):
+          self.log.debug("Input Sandbox %s is a file with absolute path, copying it" % isFile)
           shutil.copy( isFile, os.getcwd() )
-        elif os.path.exists(os.path.join(tmpdir, isFile)): # if it is in the tmp dir
-          self.log.debug("Input Sandbox %s is a file, found in the tmp directory, copying it" % isFile)
-          shutil.copy(os.path.join(tmpdir, isFile), os.getcwd())
-        elif os.path.exists(os.path.join(jobDir, isFile)): # if it is in the tmp dir
-          self.log.debug("Input Sandbox %s is a file, found in the job directory, no need to copy it" % isFile)
-        else:
-          self.log.verbose("perhaps the file %s is in an LFN, so we attempt to download it." % isFile)
-          getFile = self.getFile( isFile )
-          if not getFile['OK']:
-            self.log.warn( 'Failed to download %s with error: %s' % ( isFile, getFile['Message'] ) )
-            return S_ERROR( 'Can not copy InputSandbox file %s' % isFile )
-        basefname = os.path.basename( isFile )
+        else: # Attempt to find it and copy into job working directory
```

Comment:
Is this else with all the indentation changes necessary? Can't you just change the first `if` to `elif` for the same behaviour?
That would make the diff easier to parse